Manager/JobLauncher.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace Akeneo\Bundle\BatchBundle\Manager;
+
+use Akeneo\Bundle\BatchBundle\Entity\JobExecution;
+use Akeneo\Bundle\BatchBundle\Entity\JobInstance;
+use Doctrine\Common\Persistence\ObjectManager;
+use Pim\Bundle\ImportExportBundle\Event\JobProfileEvents;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\EventDispatcher\GenericEvent;
+use Symfony\Component\Process\PhpExecutableFinder;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @author    Marie Bochu <marie.bochu@akeneo.com>
+ * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/MIT MIT
+ */
+class JobLauncher
```

Comment:
I think this launcher will be called `SimpleJobLauncher`, to confirm with @nidup and @BitOne.